pkg/config/parameters.go

```diff
@@ -423,6 +423,15 @@ type NetworkParameters struct {
 	XffNumTrustedHops uint32 `yaml:"num-trusted-hops"`
 }
 
+// ListenerParameters hold various configurable listener values.
+type ListenerParameters struct {
+	// UseExactConnectionBalancer. If specified, the listener will use the exact connection balancer
+
+	// See https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/listener.proto#envoy-api-msg-listener-connectionbalanceconfig
+	// for more information.
+	UseExactConnectionBalancer bool `yaml:"use-exact-connection-balancer"`
```

Comment:
As I commented earlier, I think this should be called `ConnectionBalancer`, and be a string where the only valid value is `exact`. The YAML tag should be `connectionBalancer`.

Reply:
thanks @youngnick .

i agree with you.